### DISCUSSION TITLE
Messagebird Rest API calls setup file not found

### DISCUSSION THREAD

**Discussion by eloimarquessilva** [LiveHelperChat/livehelperchat#2011] [General]
Hello,
When i try to download the file configuration of messagebird API REST calls (from this link https://doc.livehelperchat.com/docs/integrating/messagebird-whatsapp/) I'm receiving a 404 Not Found.

The download link points to https://doc.livehelperchat.com/assets/files/messagebird-restapi-3c9ebf59611d82e0be668c78151d7682.json/

This file is no longer avaiable?

Thank you.

**Comment by remdex** [comment]
I'll fix. Seems after update slash was appended to links... Just temove ending slash and it will work :) There is actually no need to manually imprt it anymore, as from extension itself you can do that.

**Reply by eloimarquessilva** [reply]
Many thanks for the reply. :-)

Sorry, but which extension are you referring to?

**Reply by remdex** [reply]
https://github.com/LiveHelperChat/messagebird

**Reply by eloimarquessilva** [reply]
With this plugin can I undo the manual settings I currently have and just use the plugin? Or is the plugin just for sending templates? Thank you.

**Reply by remdex** [reply]
Just use plugin.

**Comment by remdex** [comment]
Documentation links were fixed also.
